### PR TITLE
Change one label in initialTBoxAnnotations_fr_CA.nt

### PR DESCRIPTION
**Pull request by michel-heon**
in file en_CA/home/src/main/resources/rdf/i18n/en_CA/tbox/firsttime/initialTBoxAnnotations_en_CA.nt
Change 'ExpertisesLivres' to 'Expertises'

This change affects the title of the expertise presentation in the user interface.
In French, the wording 'ExpertisesLivres' means nothing

### CODE REVIEW

File: fr_CA/home/src/main/resources/rdf/i18n/fr_CA/tbox/firsttime/initialTBoxAnnotations_fr_CA.nt [hunk 1]
```diff
@@ -609,7 +609,7 @@
 <http://www.w3.org/2006/vcard/ns#Title> <http://www.w3.org/2000/01/rdf-schema#label> "Titre"@fr-CA .
 <http://vivoweb.org/ontology/core#researchOverview> <http://www.w3.org/2000/01/rdf-schema#label> "Aperçu des recherches"@fr-CA .
 <http://purl.obolibrary.org/obo/ERO_0000390> <http://www.w3.org/2000/01/rdf-schema#label> "service fourni par"@fr-CA .
-<http://www.w3.org/2004/02/skos/core#Concept> <http://www.w3.org/2000/01/rdf-schema#label> "ExpertisesLivres"@fr-CA .
+<http://www.w3.org/2004/02/skos/core#Concept> <http://www.w3.org/2000/01/rdf-schema#label> "Expertises"@fr-CA .
```
Comment (chenejac) [inline fr_CA/home/src/main/resources/rdf/i18n/fr_CA/tbox/firsttime/initialTBoxAnnotations_fr_CA.nt:612]: I suppose this is ok, I am not a French speaker.

**Review by chenejac — COMMENTED** [review]
We need to merge only the second commit (out of three) https://github.com/vivo-project/VIVO-languages/pull/99/commits/9c571f1e1d2c0baede01dd0f16e56578958d5b22, which is change of just one triple. Is there some easier way than this one https://code-maven.com/merge-one-commit-from-a-pull-request?

**Review by wwelling — CHANGES_REQUESTED** [review]
First off, not sure why the heck there are so many pom files with the same thing in them. Maybe it is unavoidable. I suspect not though. Worth thinking about to reduce all pom files to bare minimum. I would think adding parent pom or hoisting to an existing parent pom. Anyhow, since we have to deal with these nonsensical build files for now, can we keep them generic as possible and try not to muddy up the commit history. Just kidding. I agree with Dragan, only the “language file” needs changed in this PR. Please update.

**Comment by wwelling** [comment]
I would say easiest way to recreate this PR would be to reset to head, make change in one file, commit, and force push.

**Pull request closed without merging**